### PR TITLE
Add Beckn-BPP Backend SDK & Boilerplate UI

### PR DESCRIPTION
**Pull request by aakanksha1801**
## Project

Name of the project

### Changes

What was updated (new pages, weekly update, etc.)

### Notes

If there are any further notes.

### CODE REVIEW

File: docs/2023/BeckN/Backn Smart Policy Infrastructure/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+
+# Author
+Mohammad Sarfraz Alam
+
+# Contact Info
+- [Email](mailto:alamsarfraz422@gmail.com)
+- [Linked In](https://www.linkedin.com/in/sarfraz-alam-113575201/)
+- [GitHub](https://github.com/Sarfraz-droid)
```
Comment (claddyk) [inline docs/2023/BeckN/Backn Smart Policy Infrastructure/index.md:14]: Here, you need to put your Name and 
- Email
- LinkedIn
- Github Links

File: docs/2023/BeckN/Backn Smart Policy Infrastructure/updates/2023-07-07.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Week 1
+author: Mohammad Sarfraz Alam
+---
```
Comment (claddyk) [inline docs/2023/BeckN/Backn Smart Policy Infrastructure/updates/2023-07-07.md:4]: As mentioned in my previous comment, the same goes for this part also.
Update your name here.
Also make sure you do the changes, in every week

**Comment by GeekGawd** [comment]
LGTM!

**Comment by GeekGawd** [comment]
Please keep change the slug name to be the same as folder name

**Pull request merged**